test/units/utils/test_gitlab.py

```diff
@@ -0,0 +1,559 @@
+# Copyright: (c) 2019, Guillaume Martinez (lunik@tiwabbit.fr)
```

Comment:
Move this to `test/units/modules/source_control/gitlab.py`. This will make sure that the correct unit tests are executed when changes are made to the file.

Reply:
resolved with dcdd770 and 9154899